xtask/codegen/src/language_kind.rs

```diff
@@ -56,8 +60,9 @@ impl FromStr for LanguageKind {
             "graphql" => Ok(LanguageKind::Graphql),
             "grit" => Ok(LanguageKind::Grit),
             "html" => Ok(LanguageKind::Html),
+            "yaml" | "yml" => Ok(LanguageKind::Yaml),
```

Comment:
```suggestion
            "yaml" => Ok(LanguageKind::Yaml),
```
I don't think we need to specify extension aliasses here

Reply:
That's right. This is just for internal usage.

The extensions need to be dealt towards the finish line inside biome_service